### PR TITLE
update cli for email for balsamic

### PR DESCRIPTION
**Pull request by hassanfa**
This PR update the cli for balsamic run analysis with newest version

**How to prepare for test**:
- [x] install on stage of the coffee machine: `us` and `pip install -U git+https://github.com/Clinical-Genomics/cg@balsamic_slurm_mail`
- [x] activate stage: `us`

**How to test**:
- [x] run following command: `cg analysis balsamic run --email jon.snow@spring.com --dry firmshiner `

**Expected test outcome**:
- [x] `cg` should provide with updated balsamic cli
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @hassanfa 
- [x] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is patch **version bump** because only minor change.

### CODE REVIEW

File: cg/cli/balsamic.py [hunk 1]
```diff
@@ -174,14 +174,14 @@ def run(context, dry, run_analysis, config_path, analysis_type, priority, email,
 
     # Call Balsamic
     command_str = (f" run analysis "
-                   f"--slurm-account {slurm_account}"
+                   f"--account {slurm_account}"
```
Comment (ingkebil) [inline cg/cli/balsamic.py]:
```suggestion
                   f" --account {slurm_account} "
```

**Comment by ingkebil** [comment]
Looking good! Approving when tested

**Comment by hassanfa** [comment]
Performing tests blocked, or maybe i am doing it wrong.

image

**Comment by ingkebil** [comment]
there is no `kill` in the update script, so I presume there is something weird in your bashrc

**Comment by hassanfa** [comment]
It is not `kill`. That's just trapping stuff.

There is no output printing after update-stage. I'll install it manually, it is stage after all.

**Comment by hassanfa** [comment]
Great success!

image

**Comment by ingkebil** [comment]
nice. I'll deploy

**Comment by ingkebil** [comment]
deployed

**Pull request merged**